packages/ember-engines/lib/utils/ensure-lazy-loading-hash.js

```diff
@@ -8,11 +8,15 @@
  * @return {Object}
  */
 module.exports = function ensureLazyLoadingHash(context) {
-  if (typeof context.lazyLoading === 'boolean') {
+  if (
+    typeof context.lazyLoading === 'boolean' ||
+    typeof context.lazyLoading === 'undefined'
+  ) {
     context.lazyLoading = {
-      enabled: context.lazyLoading
+      enabled: context.lazyLoading,
```

Comment:
Let's make this a bit more explicit about always being a Boolean:

```suggestion
      enabled: Boolean(context.lazyLoading),
```